flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/TestFlinkUpsert.java

```diff
@@ -171,27 +173,27 @@ public void testPrimaryKeyEqualToPartitionKey() {
     String tableName = "upsert_on_data_key";
     try {
       sql(
-          "CREATE TABLE %s(id INT NOT NULL, data STRING NOT NULL, PRIMARY KEY(data) NOT ENFORCED) "
```

Comment:
I know I suggested `name` for the test method above. I am fine with expanding the scope to all test methods. 

Let's also hear if other people have different take and think changing `data` to `name` is unnecessary here and below. I am fine either way. will approve the PR.